.github/workflows/update_api_docs.yml

```diff
@@ -0,0 +1,82 @@
+name: Update API docs
+
+on:
+  pull_request:
+  push:
+    branches:
```

Comment:
I wonder if we should trigger this workflow when commits are pushed to `develop`, what are your thoughts?

Reply:
I'm not sure what the git pattern is these days. Whatever makes sense is fine with me.